### PR TITLE
Create method for import html

### PR DESCRIPTION
**Pull request by alphp**
loadHtml parse html string and create either SimpleXmlElement object or a DOMDocument.
use of empty($options['loadEntities']) instead of !$options['loadEntities']

### CODE REVIEW

File: src/Utility/Xml.php [hunk 1]
```diff
@@ -162,7 +162,46 @@ protected static function _loadXml($input, $options)
...
+
+    /**
+     * Parse the input html string and create either a SimpleXmlElement object or a DOMDocument.
+     *
+     * @param string $input The input html string to load.
+     * @param array $options The options to use. See Xml::build()
+     * @return \SimpleXMLElement|\DOMDocument
+     * @throws \Cake\Utility\Exception\XmlException
+     */
+    public static function loadHtml($input, $options)
```
Comment (markstory) [inline src/Utility/Xml.php]: New methods require tests to be merged.

**Comment by markstory** [comment]
Moved to 3.next as new features do not go into bugfix releases.

File: src/Utility/Xml.php [hunk 2]
```diff
@@ -142,7 +142,7 @@ protected static function _loadXml($input, $options)
     {
         $hasDisable = function_exists('libxml_disable_entity_loader');
         $internalErrors = libxml_use_internal_errors(true);
-        if ($hasDisable && !$options['loadEntities']) {
+        if ($hasDisable && empty($options['loadEntities'])) {
```
Comment (markstory) [inline src/Utility/Xml.php]: >  use of empty($options['loadEntities']) instead of !$options['loadEntities']

Why though? What does this change?

Comment (alphp) [reply src/Utility/Xml.php]: If not exists $options['loadEntities'] Notice is raised:
```
2018-06-02 02:48:02 Notice: Notice (8): Undefined index: loadEntities in [C:\usr\0xFF\bambu\app\vendor\cakephp\cakephp\src\Utility\Xml.php, line 184]
```

Comment (markstory) [reply src/Utility/Xml.php]: How would that happen? The public methods set a default value.

Comment (alphp) [reply src/Utility/Xml.php]: Ok, I see.
I based loadHtml in loadXml thats is calling by build, so in loadHtml $options don't have default value.

File: tests/TestCase/Utility/XmlTest.php [hunk 3]
```diff
@@ -247,6 +247,39 @@ public function testBuildEmptyTag()
...
+     */
+    public function testLoadHtml()
+    {
+        $html = CORE_TESTS . 'Fixture/sample.html';
+        $html = file_get_contents($html);
+        $xml = Xml::loadHtml($html);
+        $xml = html_entity_decode($xml->asXML(), ENT_NOQUOTES, 'UTF-8');
+        $this->assertEquals($html, $xml);
+
+        $xml = Xml::loadHtml($html, ['parseHuge' => true]);
```
Comment (markstory) [inline tests/TestCase/Utility/XmlTest.php]: These tests are looking good. There should be a teat case covering the domdocument return tyoe as well.

**Pull request merged**